packages/core/src/repositories/documentVersionsRepository/index.ts

```diff
@@ -145,6 +144,32 @@ export class DocumentVersionsRepository extends Repository {
     return Result.ok(changedDocuments)
   }
 
+  private async getAllDocumentsAtCommit({ commit }: { commit: Commit }) {
```

Comment:
Returning all documents with deleted documents included is private. The moment we want to show then to for example "Recover from trash" we can implement `getAllDeletedDocumentsAtCommit`